src/commands/setup.rs

```diff
@@ -282,8 +161,8 @@ impl Setup {
 
             if let Err(er) = aardvark_interface.commit_netavark_entries(
                 network_options.container_name,
-                network_options.container_id,
-                network_options.networks,
+                network_options.container_id.clone(),
```

Comment:
Why do we need to clone these now.

Reply:
we have to clone it otherwise we cannot use it again
I will rework the aardvark setup next